CLAUDE.md

```
@@ -1,171 +1,115 @@
+### The data pipeline (the core mental model)
 
-Custom scripts in `src/python-tools/` define a `DEFAULTS` list for auto-generated UI:
+Every sub-app follows the same path; understanding it requires reading `src/Workflow.py`, `src/parse/`, `src/workflow/FileManager.py`, and `src/render/`:
 
-```python
-DEFAULTS = [
-    {"key": "in", "value": [], "hide": True},
-    {"key": "my-param", "value": 5, "name": "My Parameter", "help": "Description",
-     "min": 1, "max": 100, "step_size": 1, "widget_type": "slider"},
-]
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)